automated/linux/peripherals/sysfs-interface-test.yaml

```diff
@@ -0,0 +1,22 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: module-tests
+    description: |
+                 USB Gadget Framwork test
+    maintainer:
+        - anders.roxell@linaro.org
+    os:
+        - debian
+        - ubuntu
+        - openembedded
+    scope:
+        - functional
+    devices:
```

Comment:
devices:
Please add known good LAVA connected devices which supports any of these test cases.